.expeditor/update_version.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+#
+# After a PR merge, Chef Expeditor will bump the PATCH version in the VERSION file.
+# It then executes this file to update any other files/components with that new version.
+#
+
+set -evx
+
+# Temporarily fall back to cat-ing version since this change isn't in es-lita yet.
```

Comment:
sorry, not. the line should be `sed -i -r "s/VERSION = \".+\"/VERSION = \"$(cat VERSION)\"/" lib/push_client/version.rb